content/docs/command-reference/commit.md

```diff
@@ -140,83 +142,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the parameters for that script. The parameters are defined in
+the `params.yaml` file. Updating the value of `max_features` to 6000 in the
+`params.yaml` changes the resulting model:
```

Comment:
Getting better, but still there are basic grammar issues with articles in your writing. Maybe read up on their usage 🙂 

"in the `params.yaml`" is missing a noun. Articles need nouns. `params.yaml` is a name, not a noun. It could be "in the file" for example, or "in the params.yaml file" in this case.

But since we already have that exact phrase in the previous sentence, that would be repetitive so either remove "in...yaml" completely or just use "in `params.yaml` here. Try reading the sentences aloud and see which form is simpler, as long as they convey the full meaning desired.

Reply:
Yeah got it. 'the' can't be used for second time because the user already knows about the file. Thanks